packages/compile-solidity-tests/test/compileWithPragmaAnalysis.ts

```diff
@@ -129,8 +133,11 @@ describe("compileWithPragmaAnalysis", function () {
         options: config,
         paths: [path.join(sourceDirectory, "withImports", "C.sol")]
       });
+      if (!compilations) {
```

Comment:
If you're doing `if (...) { assert.fail() }`, you're not using assertions properly.  I think the right way to do this in chai would be `assert.isDefined(compilations), "Compilations were not created")`.